CMakeLists.txt

```diff
@@ -170,7 +170,7 @@ if (NOT MAKE_STATIC_LIBRARIES)
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 endif ()
 
-set (SAN_FLAGS "-O3 -g -fno-omit-frame-pointer")
+set (SAN_FLAGS "-O0 -g -fno-omit-frame-pointer")
```

Comment:
This is wrong.
The common case is to test with ASan the build with all release optimizations enabled.